platform/xbox/include/pdclib/_PDCLIB_config.h

```diff
@@ -279,7 +279,7 @@ typedef __builtin_va_list _PDCLIB_va_list;
 /* I/O ---------------------------------------------------------------------- */
 
 /* The type of the file descriptor returned by _PDCLIB_open(). */
-typedef int _PDCLIB_fd_t;
+typedef void *_PDCLIB_fd_t;
```

Comment:
nit: Why isn't this `HANDLE`?
I assume an include would drag too much stuff into this file?

Still feels a bit dirty

Reply:
> nit: Why isn't this `HANDLE`?
> I assume an include would drag too much stuff into this file?

Exactly, dragging in windows headers with every C standard library header doesn't seem like a good idea to me.

Reply:
I had originally resolved this discussion; but while doing another review pass I noticed that we should at least describe this in a comment (= that it is a `HANDLE`, and that it has to match the typedef from that header)

Reply:
I added a comment.